plugins/package-managers/python/src/funTest/assets/projects/synthetic/poetry-expected-output.yml

```diff
@@ -103,10 +103,10 @@ packages:
   description: "Safely add untrusted strings to HTML/XML markup."
   homepage_url: "https://palletsprojects.com/p/markupsafe/"
   binary_artifact:
-    url: "https://files.pythonhosted.org/packages/53/e8/601efa63c4058311a8bda7984a2fe554b9da574044967d7aee253661ee46/MarkupSafe-2.0.1-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl"
+    url: ""
```

Comment:
This diff is unexpected to me even after reading the commit message again.

Reply:
Why exactly is it unexpected? ...and what do you suggest?

Reply:
I would have expected either no change to the artifact's metadata at all, or a chnage that reflects the 3.10 -> 3.11 version upgrade, e.g. by pointing to a new artifact that does not contain "cp310" but "cp311" as part of its URL. Anyway, I do not understand why this being empty should be correct. I suggest explaining in the commit message why there is no according Python 3.11 artifact.

Reply:
I haven't figured this out myself yet. Would you be ok, if we pinned the test to version 3.10 for now and leave this open for future investigation?

Reply:
I've investigated a bit further and updated the commit message with additional info.

Reply:
If I upgrade python on docker to 3.11, then the test diff seems to be empty. 
So, this is the way to go then, @sschuberth ?

Reply:
BTW, https://pypi.org/project/MarkupSafe/2.0.1/#files shows that there's indeed no pre-built binary distribution for Python 3.11, so it's probably fine for the binary artifact to become empty, as the source artifact is still there.

As a side note, https://pypi.org/project/MarkupSafe/2.1.2/#files starts to have 3.11 binary artifacts, so if the project would upgrade to MarkupSafe 2.1.2, this should not become empty anymore.

Reply:
> BTW, https://pypi.org/project/MarkupSafe/2.0.1/#files shows that there's indeed no pre-built binary distribution for Python 3.11, so it's probably fine for the binary artifact to become empty, as the source artifact is still there.

This is almost exactly what the commit message now says.

Reply:
> If I upgrade python on docker to 3.11, then the test diff seems to be empty.
> So, this is the way to go then, @sschuberth ?

This turned out to be wrong. A mistake during debugging.